### ISSUE TITLE
Broken link to Quarkus documentation for S2I Build

### ISSUE THREAD

**Issue by AlyIbrahim** [quarkusio/quarkus-images#106]
https://quarkus.io/guides/deploying-to-openshift-s2i

This link from README at ./quarkus-images/modules/quarkus-native-s2i-scripts.

This page was at Quarkus website before but I couldn't find it in the guides anymore.

**Comment by cescoffier** [comment]
@geoand By any chance, do you know where it is now? It seems that the content has been replaced with the built-in feature, but that does not cover the _regular_ s2i build.

**Comment by geoand** [comment]
I am pretty sure we removed it because we had just too many to deploy things to OpenShift and it was confusing for users...

**Comment by cescoffier** [comment]
so, we don't have the instructions to run a regular s2i anymore? That is a bit annoying.

**Comment by geoand** [comment]
Exactly. Not sure what we should do about it... The problem before was that with too many options people where getting confused...

**Comment by AlyIbrahim** [comment]
I think it would be better if we can bring it back, may be restructuring the builds but I feel this info is important to be out

**Comment by cescoffier** [comment]
@AlyIbrahim We are going to restore it. We are investigating to see what has been deleted and where it should be restored.

**Comment by cescoffier** [comment]
ping @maxandersen - I know it's a long time, but did you do anything?

**Comment by maxandersen** [comment]
old issue - but i see we have https://quarkus.io/guides/deploying-to-openshift-s2i-howto so going to close this ;)
